homeassistant/components/esphome/entry_data.py

```diff
@@ -159,7 +161,7 @@ async def async_save_to_store(self) -> None:
         for service in self.services.values():
             store_data["services"].append(service.to_dict())
 
-        await self.store.async_save(store_data)
```

Comment:
Nice one.

We should find all the `async_save` and kill them unless it's auth related (user/config entry).

Reply:
I'll audit what else is done at startup and adjust since these are the ones we except to be I/O bound since they are competing with loading the `.py` files from disk.